### ISSUE TITLE
Make a reviewer deadline

### ISSUE THREAD

**Issue by eteq** [openjournals/joss#312]
Sorry if this isn't the right place to make a "process" issue, but it felt right...

Perhaps JOSS should have a reviewer deadline?  Many of the academic journals I'm used to set a clock (2 weeks, 4 weeks, etc.) from when the reviewer accepts.  Of course we all know these aren't "real" deadlines in that usually they are flexible... but having the deadline has a psychological impact, at least for some reviewers, that makes them do it in a timely (or at least predictable) manner.

**Comment by karthik** [comment]
Thanks @eteq We typically give people 3 weeks for a review but it would be good to note that in the reviewer guidelines.

**Comment by danielskatz** [comment]
I actually disagree with this.  I think most reviewers take a deadline of x weeks to mean, good, I can wait x weeks before I look at this.  Right now, JOSS reviews generally happen more quickly, and the weekly reminders are very effective for editors, who (at least in my case) use them to remind reviewers.  I think this is part of what makes our process a lot faster that most journals.

But @eteq, this is certainly the right place to make a process issue and to have a discussion about - it's good you brought this up here.

**Comment by eteq** [comment]
Hmm, I see your point, @danielskatz, but that says to me that the "deadline" should be "1 week"?  In my (admittedly limited) reviewer experience, that's realistic, but right now my initial reaction is "no deadline?  I can just do it whenever then." and "whenever" has a way of being "the undefined future" if you get busy.  But if it said "1 week", I'd probably have done both of those I've reviewed thus far within that week.

Of course it should really be "1 week unless you say otherwise".  Then I can say "Oh, I'm on vacation this week, but can do it a week after I get back".  Then it goes on my todo list and I'm more likely to actually do it... all without *any* editor interaction needed.

And I think the speed is more because the review process itself is relatively *easy* relative to a paper submission, in substantial part due to the clear guidelines.

**Comment by labarba** [comment]
I agree that mentioning some deadline is a good reference point for the reviewer—and I agree that if this deadline is generous, the reviewer will do nothing until it approaches. The solution is to give a tight (but negotiable) deadline. I propose we say "two weeks."

**Comment by jmjwozniak** [comment]
If there is no review deadline, I would make that clear under "Reviewer questions" in the main review issue.

**Comment by danielskatz** [comment]
I like 2 weeks too.
